regions/webview/content_item.py

```diff
@@ -8,7 +8,7 @@
 
 
 class ContentItem(Region):
-    _chapter_section_span_locator = (By.CSS_SELECTOR, "span.chapter-number")
+    _chapter_section_span_locator = (By.CSS_SELECTOR, "span.os-number")
```

Comment:
Just curious... What does `os` mean here? I understood what `chapter` indicated, but I'm at a loss with `os`.

Reply:
Not sure. They decided to change it to os-number. i'm assuming "os" is OpenStax. Maybe they wanted it to be more generic in the recipes?